From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Antonio Vieiro <avieirov@redhat.com>
Date: Mon, 3 Jun 2024 10:35:25 +0000
Subject: [PATCH] 8240226: DeflateIn_InflateOut.java test incorrectly assumes
 size of compressed file

Backport-of: b38f3cf3bdde596c07c00106e8a60263ee6cb374
---
 test/jdk/java/util/zip/DeflateIn_InflateOut.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/jdk/java/util/zip/DeflateIn_InflateOut.java b/test/jdk/java/util/zip/DeflateIn_InflateOut.java
index dd69a7773eb..bce10c30b05 100644
--- a/test/jdk/java/util/zip/DeflateIn_InflateOut.java
+++ b/test/jdk/java/util/zip/DeflateIn_InflateOut.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2006, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2006, 2020, Oracle and/or its affiliates. All rights reserved.
  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
  *
  * This code is free software; you can redistribute it and/or modify it
@@ -42,8 +42,6 @@ public class DeflateIn_InflateOut {
     private static InflaterOutputStream ios;
 
     private static void reset() {
-        new Random(new Date().getTime()).nextBytes(data);
-
         bais = new ByteArrayInputStream(data);
         dis = new DeflaterInputStream(bais);
 
@@ -218,6 +216,8 @@ public class DeflateIn_InflateOut {
 
 
     public static void realMain(String[] args) throws Throwable {
+        new Random(new Date().getTime()).nextBytes(data);
+
         ArrayReadWrite();
 
         ArrayReadByteWrite();
